src/ducks/categories/CategoriesPage.jsx

```diff
@@ -248,43 +254,94 @@ const enhance = Component => props => {
   const period = useSelector(getPeriod)
   const dispatch = useDispatch()
 
-  const initialConn = makeFilteredTransactionsConn({
+  const initialConnByDate = makeFilteredTransactionsConn({
+    groups,
+    accounts,
+    filteringDoc,
+    dateAttribute: 'date'
+  })
+  const initialConnByApplicationDate = makeFilteredTransactionsConn({
     groups,
     accounts,
-    filteringDoc
+    filteringDoc,
+    dateAttribute: 'applicationDate'
```

Comment:
Shouldn't we put these strings (date, applicationDate, maybe there are others) in constants ? Would feel a bit safer